### ISSUE TITLE
about loss function

### ISSUE THREAD

**Issue by laisimiao** [joshuajss/RTAA#2]
hi,  @joshuajss Could I ask you some questions about loss function part:
https://github.com/joshuajss/RTAA/blob/e9fed9d6c8626a1646258dfc81edfee5d71f109d/DaSiamRPN/code/run_attack.py#L138-L146
When we calculate cls loss anf reg loss, why need to take 'minus' like above line139-140 and line144-145,  I guess the reason is: 
https://github.com/joshuajss/RTAA/blob/e9fed9d6c8626a1646258dfc81edfee5d71f109d/DaSiamRPN/code/run_attack.py#L159
we use gradient descent rather than gradient ascent(as paper), am I right?

**Comment by joshuajss** [comment]
It is OK. In line 139-140, we refer to the code of FGSM.

**Comment by laisimiao** [comment]
Oh, I see. I have not yet learned about FGSM, thanks.
